pkgs/top-level/all-packages.nix

```diff
@@ -12296,13 +12296,6 @@ in
   php74Extensions = recurseIntoAttrs php74.extensions;
   php74Packages = recurseIntoAttrs php74.packages;
 
-  # Import PHP73 interpreter, extensions and packages
-  php73 = callPackage ../development/interpreters/php/7.3.nix {
```

Comment:
I think we should add an alias for this to prevent eval errors.

Reply:
Added to aliases :slightly_smiling_face: